### PR TITLE
CE-796 - Mobile refactor - Assessment, Prevention & Education and Notes

### PR DESCRIPTION
**Pull request by jeznorth**
# Description

Updating the read only and form components for the following Outcome Report sections:
- Complaint Assessment
- Prevention and Education
- Additional Notes
- Updated the basic layout structure for other Outcome Report components so the layout doesn't appear broken.

Fixes # (issue)

# How Has This Been Tested?

- [x] Manually tested
- [x] Ran cypress tests associated with each component (where needed)

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Further comments

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-compliance-enforcement-457-frontend.apps.silver.devops.gov.bc.ca/api/) available
[Frontend](https://nr-compliance-enforcement-457-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge-main.yml)

---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend](https://nr-compliance-enforcement-457-frontend.apps.silver.devops.gov.bc.ca/api/) available
[Frontend](https://nr-compliance-enforcement-457-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/nr-compliance-enforcement/actions/workflows/merge-main.yml)

### CODE REVIEW

File: frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx [hunk 1]
```diff
@@ -297,197 +297,195 @@ export const HWCRComplaintAssessment: FC = () => {
...
-                    formatDate(`${selectedDate}`)
-                  )}
-                </div>
-              </div>
+          </div>
+          <div
+            id="justification-div"
+            className="comp-details-form-row"
+          >
+            <label
```
Comment (afwilcox) [inline frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx:363]: The Justification field should only show when Action Required is set to no.

**Review by afwilcox — CHANGES_REQUESTED** [review]
In addition to the comments in the code, I believe all the SonarCloud Issues should be able to be resolved.

File: frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx [hunk 2]
```diff
@@ -297,197 +297,195 @@ export const HWCRComplaintAssessment: FC = () => {
...
+              className={justificationLabelClass}
+              htmlFor="justification"
+            >
+              Justification
```
[the remark this answers is not in the document]
Comment (jeznorth) [reply frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx]: Updated and tested locally. Should be resolved.

File: frontend/src/app/components/containers/complaints/outcomes/hwcr-prevention-education.tsx [hunk 3]
```diff
@@ -226,152 +226,153 @@ export const HWCRComplaintPrevention: FC = () => {
...
-                  </div>
+        <>
+          {editable ? (
+            <div className="comp-details-form">
+              <div className="comp-details-form-group">
+                <div
+                  id="prev-educ-checkbox-div"
+                  className="comp-details-form-row"
+                >
+                  <label htmlFor="checkbox-div">Actions</label>
```
[the remark this answers is not in the document]
Comment (jeznorth) [reply frontend/src/app/components/containers/complaints/outcomes/hwcr-prevention-education.tsx:268]: Updated and tested locally. Should be resolved.

File: frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx (hunk 1 above)
Comment (jeznorth) [reply frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx]: Updated to provide accurate display of form field here.

**Comment by jeznorth** [comment]
Breaking this into smaller chunks for pull requests.

**Pull request closed without merging**